readthedocs/templates/builds/build_detail.html

```diff
@@ -138,9 +138,14 @@
         {% trans "Build completed" %}
       </span>
       <span class="build-state-failed"
-            data-bind="visible: finished() && !success()"
+            data-bind="visible: finished() && !success() && !cancelled()"
```

Comment:
This is the type of logic that I'll move to strictly JS and would like out of templates eventually. For now this is fine though.